pkg/utils/reader.go

```diff
@@ -121,6 +223,28 @@ func (r *DynamicUserGenerator) generateRandomBody() {
 	randomizeBody(r.faker, r.baseBody, r.ts)
 }
 
+func (r *K8sGenerator) createK8sBody() {
```

Comment:
you can put all of the content of the function `createK8sBody` right here and not need the func defined in line #128 and #114